src/Configurator/DockerComposeConfigurator.php

```diff
@@ -128,6 +132,73 @@ public function unconfigure(Recipe $recipe, $config, Lock $lock)
         $this->write('Docker Compose definitions have been modified. Please run "docker-compose up" again to apply the changes.');
     }
 
+    public static function shouldConfigureDockerRecipe(Composer $composer, IOInterface $io, Recipe $recipe): bool
+    {
+        if (null !== self::$configureDockerRecipes) {
+            return self::$configureDockerRecipes;
+        }
+
+        $dockerPreference = $composer->getPackage()->getExtra()['symfony']['docker'] ?? null;
+        if (null !== $dockerPreference) {
+            self::$configureDockerRecipes = $dockerPreference;
+
+            return self::$configureDockerRecipes;
+        }
+
+        if ('install' !== $recipe->getJob()) {
+            // default to not configuring
+            return false;
+        }
+
+        $warning = $io->isInteractive() ? 'WARNING' : 'IGNORING';
+        $io->writeError(sprintf('  - <warning> %s </> %s', $warning, $recipe->getFormattedOrigin()));
+        $question = '    The recipe for this package contains some Docker configuration.
+
+    This may create/update <comment>docker-compose.yml</comment> or update <comment>Dockerfile</comment> (if it exists).
```

Comment:
```suggestion
    This may create/update <comment>docker-compose.yml</comment> or update <comment>Dockerfile</comment> (if they exist).
```
?

Reply:
It may also create/update the `docker-compose.override.yml` file, not sure if adding it in the list is useful though.

Reply:
I used "if it exists" because this only applies to `Dockerfile`. The `docker-compose.yml` is already created automatically for you even if it doesn't exist.

> It may also create/update the docker-compose.override.yml file, not sure if adding it in the list is useful though.

I had the same thought - decided to not mention it in the list for now.